### ISSUE TITLE
eusebian canons

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#336]
I see that the ID LIT1224Canons has been deleted and supplanted by LIT1560Gospel#IntroductionGospels

During the last meeting however I got the idea that the Canons should be subjected to a detailed analysis and therefore cannot become a textPart - we should moreover further distinguish between the sets of 1 letter + 8 canon tables or 3 letters + 7 canon tables (distinct types)

There are also still records that use the ID - before deleting IDs it must be made sure that no record points to it.

**Comment by thea-m** [comment]
True, the canons are now actually LIT1560Gospel#IntroductionCanons, though the old LIT1224Canons  hasn't been deleted yet (despite being struck out in the spreadsheet)
How should the work IDs be (re)created? 
- LIT1224Canons as a generic record if no more specific information is available from the catalogues
- An ID for  "1 letter + 8 canon tables" (with relations to individual IDs of the letter and each canon table, or with textparts?) 
- An ID for "3 letters + 7 canon tables" (with relations to individual IDs of the letters and each canon table, or with textparts?) 

?
@abausi @MassimoVilla @DenisNosnitsin1970

**Comment by abausi** [comment]
I think that this is OK: but there should also be additional IDs (to be 
applied of course when the information is available), for each of the 
varying typologies. If I remember well, there are also models with more 
canon tables.

(Consider these contributions here attached.)

Alessandro B.

Il 22.02.2017 12:06, thea-m ha scritto:

**Comment by thea-m** [comment]
I'm sorry, I can't see any attachments...

**Comment by abausi** [comment]
Il 24.02.2017 11:48, thea-m ha scritto:

**Comment by MassimoVilla** [comment]
I have the same problem as @thea-m. I can not see them even in the reply.

**Comment by MassimoVilla** [comment]
Perhaps we could be able to see the attachments via regular e-mail.

**Comment by abausi** [comment]
Il 24.02.2017 12:00, MassimoVilla ha scritto:

**Comment by abausi** [comment]
Did you receive?

Il 24.02.2017 12:03, Alessandro Bausi ha scritto:

**Comment by MassimoVilla** [comment]
I received them in the email which is automatically sent to my address together with the message here.
Thank you. I will send to @thea-m

**Comment by abausi** [comment]
(Ho mandato anche a Thea, dovrebbe aver ricevuto.)

Il 24.02.2017 12:09, MassimoVilla ha scritto:

**Comment by PietroLiuzzo** [comment]
Please close if completed

**Comment by abausi** [comment]
Concerning the Canons and Letters: the (most ancient) patterns are: 2 pages of letter (not 2 letters) + 8 pages of canons OR 3 pages of letter + 7 pages of canons. So please, consider thess arrangements.

**Comment by nafisa-valieva** [comment]
Bonjour!
Being re-adressed by Eugenia to this issue, I do not feel relieved at all. I am trying to catch up, but for now it is very confusing as it is. 
(What professor Bausi noted in his last comment seems to be related to the manuscripts rather, how the work was realised in it.)
Do I understand right that the idea is to create a different work ID for each iconographic type, according to the typology? I would ask Jacobo for his opinion for this. (I do not know how to invite him here.) 
The discussion was left without any conclusion.
So,  being in agreement with the statment that 
"We accept as a principle to create the maximum number of record as it is sensible and to state the minimum number of relations needed among them with a view on expanding them potentially to all what is not essential, in order to clarify their relation with each other.", 
I still find that in the present form various records are confusing, at least incipits and explicits would help, otherwise we (me!) are running into serious problems. 
To sum up my confusion: how I am supposed to decide which ID I should assign, LIT1349EpistlEusebius
LIT1224Canons
or LIT4876LetterCanons: Letter to Carpianus and Eusebian Canons?
And from the discussion I read that there is even LIT1560Gospel#IntroductionGospels
, which was meant to supplement  LIT1224Canons? 
If no one has time to solve this, I will use
LIT1349EpistlEusebius
LIT1224Canons
and neglect the rest, ok?

**Comment by Gnisci** [comment]
Hi,

Let me see if I can provide some inputs on this question.

First the Epistle and Canons need to be treated as two separate entities. They are often associated to each other, but not necessarily so. You may have the Canons without the Epistle (I've never come across the opposite). 

The Epistle has nothing to do with LIT1560Gospel which is = to LIT4872MaqdemaWangel, that is a different text.

Finally, both the Epistle and the Canon can be decorated with frames. The information concerning the frames should be encoded following the schema that Thea and Denis have been working on, discussed here https://github.com/BetaMasaheft/Documentation/issues/888

**Comment by abausi** [comment]
Sorry, I cannot follow all this now, but just a couple of basic indications.

The fundamental question has been evoked by Nafisa.

Minimal distinction is obviously between the Epistle and the Canons. 
Yet, for the arrangement of the canon, that is, which canons of the 10, 
in how many pages (e.g. 2 Letter + 8 Canons or 3 Letter + 7 Canons as in 
the two AG MSS), predetermined types could also be considered.

The question of the "Prefatory material to the Gospel" is also complex, 
because "Maqdǝma Wangel", extensively, could also include "Letter of 
Eusebius" and "Canons". It is essential here to refer to some authority 
on which to base these distinctions and identifications: obviously, the 
article by Cowley (Roger W. Cowley, ‘New Testament Introduction in the 
Andemta Commentary Tradition’, Ostkirchliche Studien, 26 (1977), 
144–192), and the Introduction to the edition of Mark by Zuurmond.

Il 21.11.2019 18:49, Jacopo Gnisci ha scritto:

**Comment by nafisa-valieva** [comment]
Thank you very much!!
 From what  can see, we all agree on:
 LIT1349EpistlEusebius
LIT1224Canons
LIT4872MaqdemaWangel
For various arrangements as many ID could be created, but these IDs need to be specified explicitly then:  LIT4876LetterCanons can be modified into one ID.
Then, in a given manuscript we can have
LIT4872MaqdemaWangel+
LIT1224Canons+LIT1560Gospel 
 LIT1349EpistlEusebius+LIT1224Canons+LIT1560Gospel 
etc

No need for Works ID from each iconographic type
Since the works are destined for the editions, since the edition of LIT1349EpistlEusebius exists, Pietro can easily convert it into xml, (or may be I can do it, if professor Bausi will give me everything I need)
Then I do not understand 
Then, LIT2705Introd contains MaqdemaWangel and LIT1349EpistlEusebius
LIT1224Canons? 
"Yet, for the arrangement of the canon, that is, which canons of the 10,
in how many pages (e.g. 2 Letter + 8 Canons or 3 Letter + 7 Canons as in
the two AG MSS), predetermined types could also be considered. " - if the content is the same and we are dealing rather with mise en page, I wonder whether two works ids are reasonable in this case, i.e. to have  LIT4876LetterCanons and LIT4677LetterCanons as types? This is something to be noted rather in each manuscript, where one can also identify a type, referring to literature. 
 Should I refer to LIT1349EpistlEusebius and indicate as locus ff. 2r-3r (which makes 3pages!) and to LIT1224Canons and indicate as locus ff. 3v-6v (which makes 7pages) or instead, refer to  LIT4877LetterCanons and indicate as locus ff. 2r-6v? In my opinion the first variant is much better, so I wonder why we should keep confusing LIT4876LetterCanons and LIT4677LetterCanons.

I think this issue should remain open, until we do not find all time to have a reasonable good decision. 
have a nice weekend!

**Comment by abausi** [comment]
No time to answer in detail

Il ven 22 nov 2019 18:39 nafisa-valieva <notifications@github.com> ha
scritto:

**Comment by thea-m** [comment]
Thank you @nafisa-valieva and @Gnisci for taking the time for this discussion!

(And see also https://github.com/BetaMasaheft/Documentation/issues/459)

Just a few general remarks on the more minor issues involved in this, as I have no time now to get deeper into this issue.

- The proposed example structure 

> LIT4872MaqdemaWangel+
> LIT1224Canons+LIT1560Gospel
> LIT1349EpistlEusebius+LIT1224Canons+LIT1560Gospel

is completely fine, since LIT1224Canons and LIT1349EpistlEusebius are "generic records", which are created for works that have more than one recension/version and can be used in case of uncertainty of the precise version contained in a manuscript.

- The edition of LIT1349EpistlEusebius is already present: https://betamasaheft.eu/works/LIT1349EpistlEusebius/text

- Yes, LIT2705Introd contains the other introductory records, as stated in https://github.com/BetaMasaheft/Documentation/issues/459

- > LIT1224Canons?

I'm sorry, I don't understand the question :) Do you mean why it is contained in LIT2705Introd ?

- Regarding the main issue behind all this, whether the two records LIT4876LetterCanons and LIT4677LetterCanons should exist and be used in the description of manuscripts or whether the difference, which is mainly one in layout, should be described only in the manuscripts , should possibly be discussed with @abausi (who proposed this structure, but of course it can be reconsidered in view of the development of our encoding and concepts in the last years). 

-  > Should I refer to LIT1349EpistlEusebius and indicate as locus ff. 2r-3r (which makes 3pages!) and to LIT1224Canons and indicate as locus ff. 3v-6v (which makes 7pages) or instead, refer to LIT4877LetterCanons and indicate as locus ff. 2r-6v? 

According to the current state of records, both solutions are possible, though the second one would be preferable. But if you have persisting doubts of the usefulness of the existence of LIT4877LetterCanons  and want to proceed with your encoding before this issue is closed, by all means go on with the first possibility.
